src/components/Settings/SettingsPlex.tsx

```diff
@@ -52,12 +62,143 @@ interface SyncStatus {
   libraries: Library[];
 }
 
+interface PresetServerDisplay {
+  name: string;
+  ssl: boolean;
+  uri: string;
+  address: string;
+  host: string;
+  port: number;
+  local: boolean;
+}
 interface SettingsPlexProps {
   onComplete?: () => void;
 }
 
+function PlexPresetOption(props: { key: string; server: PresetServerDisplay }) {
+  if (props.server.local) {
+    return (
+      <option key={props.server.uri} value={props.server.uri}>
+        {' '}
+        {props.server.name}
+        {' ('}
+        {props.server.address}
+        {') [local]'}
```

Comment:
Use a template literal here to clean this up (and the one below it). Also, instead of having an if statement to render two different options depending on the state of local, do it all in one line and drop the if statement.
```suggestion
        {`${server.name} (${server.address}) [${server.local ? 'local' : 'remote'}]`}
```